Doc/library/subprocess.rst

```diff
@@ -1150,6 +1150,11 @@ calls these functions.
    :exc:`CalledProcessError` object will have the return code in the
    :attr:`~CalledProcessError.returncode` attribute.
 
+   If :func:`check_call` was unable to start the process it will return the error
```

Comment:
Will it return an error? or propagate whatever exception was raised?

Reply:
Actually I just notice there is duplication between this paragraph and the previous one.

Reply:
Shall we leave the rephrased version if it is more obvious?

Reply:
I think the original is more precise w.r.t the wrapped returncode value.

I would leave the original paragraph and just append to it a sentence or half-sentence about what happens if the child process could not even start. This order seems more natural to me: the common case is your job succeeded, then that your job failed, then the least common case is that your job could not even launch.

Reply:
I've added the changes